content/about.md

```diff
@@ -1,5 +1,5 @@
 Hello, another time! My name is Dmitriy and I am 22 y.o. software engineer from Ukraine. I've started my career when I was 17 as a trainee .Net developer. But then I tried javascript and absolutely fall in love with front-end and javascript community at all. Then I continued my work as a JavaScript engineer in the big outsource company, started as a developer and became Senior/Tech lead.
 
-By the way, I am a fan of open-source. Now I am the collaborator of [material-ui](https://material-ui.com/discover-more/team/) - the most popular UI kit for React in the world. Creator + maintainer of [material-ui-pickers](https://material-ui-pickers.dev/) - set of date/time pickers for Material-UI. Also member of react-native-community and maintainer of [react-native-status-bar](https://github.com/react-native-community/react-native-statusbar). I love to make the software world better.
+By the way, I am a fan of open-source. Now I am a collaborator of [Material-UI](https://next.material-ui.com/discover-more/team/) - the most popular UI kit for React in the world. Creator + maintainer of [material-ui-pickers](https://material-ui-pickers.dev/) - set of date/time pickers for Material-UI. Also member of react-native-community and maintainer of [react-native-status-bar](https://github.com/react-native-community/react-native-statusbar). I love to make the software world better.
 
-In short: speaker, holistic open-source contributor. Typescript/Ocaml, react and react-native fan. Also volleyball player and [amateur photographer](https://www.instagram.com/dmtr._.kovalenkoo/) (ᵔᴥᵔ)
\ No newline at end of file
+In short: speaker, holistic open-source contributor. Typescript/Ocaml, react and react-native fan. Also volleyball player and [amateur photographer](https://www.instagram.com/dmtr._.kovalenkoo/) (ᵔᴥᵔ)
```

Comment:
No idea why this appears as a change. Didn't touch that section.

Reply:
Seems like a prettier :) No worries. Thank you a lot. I just didn't check it